src/mlpack/core/util/version.hpp

```diff
@@ -18,7 +18,7 @@
 // with higher number than the most recent release.
 #define MLPACK_VERSION_MAJOR 4
 #define MLPACK_VERSION_MINOR 5
-#define MLPACK_VERSION_PATCH 1
+#define MLPACK_VERSION_PATCH 2
```

Comment:
Why are we setting it to 2 here when it is supposedly 4.5.1 ?

Reply:
Gotcha. I was about to add 'or maybe I do not understand the local process well enough' so it is HEAD~1 for the release and this is one ahead. (I tend to split those. Oh well.)

"Not obvious, though." as the kids say.  Thanks for the clue bat wave.